src/main/webapp/assets/readfile.css

```diff
@@ -1,5 +1,9 @@
 body {
+  background-image: linear-gradient(to right,rgb(220, 220, 220),rgb(169, 169, 169));
+  background-size: cover;
+  color:black;
```

Comment:
nit: Needs a space in between the colon and black

Reply:
nice catch, thanks